src/components/GlobalStyles.tsx

```diff
@@ -1,82 +1,83 @@
+import { css } from '@emotion/core'
 import { colors, fonts, getCdnFontFaces } from '@hedviginsurance/brand'
-import * as React from 'react'
-import { injectGlobal } from 'react-emotion'
 
-export const GlobalStyles: React.FunctionComponent = () => {
-  // tslint:disable-next-line no-unused-expression
-  injectGlobal`
-${getCdnFontFaces()}
+export const globalStyles = () => css`
```

Comment:
Does this need to be a function? 🙂

Reply:
no, but i like the immutability

Reply:
Will it not be evaluated once at module evaluation time and then be constant?

Reply:
yes, but you can still modify it after. I guess its a matter of preference, i like the thought of not being able to modify stuff. But i can make it mutable

Reply:
It's not super-important to me. I think as a rule if someone starts mutating the output of a module we have to bust them in PRs anyways as that's highly illegal 👮

Reply:
haha yes that's fair